components/authentication/managed-workspaces.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
This file should be added in `kustomization.yaml` too, no?

Reply:
yep! good catch

Reply:
fixed